apisix/plugins/limit-count/limit-count-redis-cluster.lua

```diff
@@ -91,16 +92,18 @@ function _M.incoming(self, key)
     local window = self.window
     key = self.plugin_name .. tostring(key)
 
-    local remaining, err = red:eval(script, 1, key, limit, window)
+    local res, err = red:eval(script, 1, key, limit, window)
+    local remaining = res[1]
```

Comment:
Please use the res after checking the err

Reply:
fixed